app/Resources/static/themes/_global/js/tools.js

```diff
@@ -1,5 +1,7 @@
 const $ = require('jquery');
 
+var jrQrcode = require('jr-qrcode');
```

Comment:
No `var`, use `const` instead.
But you better go with es6 imports like this : `import jrQrcode from 'jr-qrcode`